code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -346,16 +346,16 @@
 /obj/item/weapon/gun/projectile/automatic/carbine
 	name = "assault carbine"
 	desc = "The bullpup configured GP3000 is a lightweight, compact, military-grade assault rifle produced by Gurov Projectile Weapons LLC. It is sold almost exclusively to standing armies. The serial number on this one has been scratched off. Uses 5.56mm rounds."
-	icon_state = "bullpupm"
+	icon_state = "bullpup"
 	item_state = "bullpup"
 	w_class = ITEMSIZE_LARGE
 	force = 10
-	caliber = "a556"
+	caliber = "a762"
 	origin_tech = list(TECH_COMBAT = 6, TECH_MATERIAL = 1, TECH_ILLEGAL = 4)
 	slot_flags = SLOT_BACK
 	load_method = MAGAZINE
-	magazine_type = /obj/item/ammo_magazine/a556m
-	allowed_magazines = list(/obj/item/ammo_magazine/a556, /obj/item/ammo_magazine/a556m)

```

Comment:
You also broke this gun's ability to hold smaller or larger magazines entirely. On the one hand you pulled it out of the admin abuse pit but on the other hand you also made the icons completely inconsistent.